hi-ml-multimodal/notebooks/phrase_grounding.ipynb

```diff
@@ -51,7 +51,7 @@
    },
    "outputs": [],
    "source": [
-    "repo_branch = \"main\""
+    "pip_source = \"hi-ml-multimodal==0.1.0\""
```

Comment:
I think we could leave the package version unpinned here - then users will get the latest version automatically and we don't have to update this every release. We have a test for this notebook, so using the latest version by default should be fine.

Reply:
```suggestion
    "pip_source = \"hi-ml-multimodal\""
```